invenio/ext/script/__init__.py

```diff
@@ -76,8 +80,8 @@ def check_for_software_updates(flash_message=False):
         find = re.compile('Invenio v[0-9]+.[0-9]+.[0-9]+(\-rc[0-9])?'
                           ' is released')
 
-        webFile = urllib.urlopen("http://invenio-software.org/repo"
-                                 "/invenio/tree/RELEASE-NOTES")
+        webFile = urllib.request.urlopen("http://invenio-software.org/repo"
+                                         "/invenio/plain/RELEASE-NOTES")
 
```

Comment:
I'd point to GitHub, for example: 

- https://raw.githubusercontent.com/inveniosoftware/invenio/master/RELEASE-NOTES